wheel/src/run_generator.rs

```diff
@@ -42,52 +42,48 @@ pub struct PySpendBundleConditions {
     pub seconds_absolute: u64,
     // Unsafe Agg Sig conditions (i.e. not tied to the spend generating it)
     #[pyo3(get)]
-    pub agg_sig_unsafe: Vec<(PyObject, PyObject)>,
+    pub agg_sig_unsafe: Vec<(Bytes48, Bytes)>,
     #[pyo3(get)]
     pub cost: u64,
 }
 
-fn node_to_pybytes(py: Python, a: &Allocator, n: NodePtr) -> PyObject {
-    PyBytes::new(py, a.atom(n)).into()
-}
-
-fn convert_spend(py: Python, a: &Allocator, spend: Spend) -> PySpend {
-    let mut agg_sigs = Vec::<(PyObject, PyObject)>::new();
+fn convert_spend(a: &Allocator, spend: Spend) -> PySpend {
+    let mut agg_sigs = Vec::<(Bytes48, Bytes)>::new();
     for (pk, msg) in spend.agg_sig_me {
-        agg_sigs.push((node_to_pybytes(py, a, pk), node_to_pybytes(py, a, msg)));
+        agg_sigs.push((a.atom(pk).into(), a.atom(msg).into()));
     }
-    let mut create_coin = Vec::<(PyObject, u64, PyObject)>::new();
+    let mut create_coin = Vec::<(Bytes32, u64, Option<Bytes>)>::new();
     for c in spend.create_coin {
         create_coin.push((
-            PyBytes::new(py, &c.puzzle_hash).into(),
+            c.puzzle_hash,
             c.amount,
-            node_to_pybytes(py, a, c.hint),
+            if c.hint != a.null() {
+                Some(a.atom(c.hint).into())
+            } else {
+                None
+            },
         ));
     }
 
     PySpend {
-        coin_id: PyBytes::new(py, &*spend.coin_id).into(),
-        puzzle_hash: node_to_pybytes(py, a, spend.puzzle_hash),
+        coin_id: *spend.coin_id,
+        puzzle_hash: a.atom(spend.puzzle_hash).into(),
         height_relative: spend.height_relative,
         seconds_relative: spend.seconds_relative,
         create_coin,
         agg_sig_me: agg_sigs,
     }
 }
 
-fn convert_spend_bundle_conds(
-    py: Python,
-    a: &Allocator,
-    sb: SpendBundleConditions,
-) -> PySpendBundleConditions {
+fn convert_spend_bundle_conds(a: &Allocator, sb: SpendBundleConditions) -> PySpendBundleConditions {
```

Comment:
these type no longer contain python-allocated types (like `PyBytes`), so we don't need a `Python` object to allocate them